docs/data-sources/kubernetes_secret.md

```diff
@@ -10,20 +10,21 @@ Manage a Kubernetes Secret using this Terraform module.
 
 In managed clusters, both attached and provisioned, you can create Kubernetes Secret that you can manage through Tanzu Mission Control.
 
-To create a cluster secret, you must be associated with the cluster.admin role
+To create a kubernates secret, you must be associated with the cluster.admin or clustergroup.admin role.
```

Comment:
Fix this typo here and throughout the docs

Reply:
Updated